agents/chart/main.py

```diff
@@ -2,23 +2,38 @@
 
 import fixieai
 
-BASE_PROMPT = """I am an agent that makes sick charts."""
+BASE_PROMPT = """I am an agent that makes beautiful charts using the Quickchart API."""
 
 FEW_SHOTS = """
 Q: Make a chart of the 5 tallest mountains
-Thought: I need to make a JSON object with information about the 5 tallest mountains.
+Thought: I need to ask the router agent for the 5 tallest mountains and their elevations.
+Ask Agent[router]: What are the 5 tallest mountains and their elevations?
```

Comment:
Would it not be better to just ask `Ask Agent[search]` directly? Faster and less likely to fail

Reply:
This is a bad fit for search, GPT will have better knowledge of this data than search. We could use search for some real-time data, but even there we would need some chain strategy to get the data in a form that we can actually use (which the model is happy to do but is more complicated with search results).

Reply:
No, because web search is not always the right place to get the data. Try "Make a bar chart of the top 10 contributors to the fixie-ai/fixie github repo" for example.